SEP-0002.md

```diff
@@ -135,6 +135,12 @@ present, and such other information as shall be necessary to determine
 the actions taken and whether the meeting was held in accordance with
 the law and these Bylaws.
 
+* **Chief Financial Officer**: The board shall select a Chief Financial Officer
+(CFO). The CFO cannot also serve as the Executive Director. The CFO shall
+be responsible for tracking and reporting of organizational assets to the board.
+The CFO shall also be responsible for releasing funds, as needed, with the
```

Comment:
CFO should be able to approve spending without needing to find two other members.  Or at least up to a reasonable dollar amount.  The fact that we are an actual organization should make there be sufficient penalties for mismanaging funds and such.

Then each board meeting should have a brief financial report, handed in advance or at least as an agenda item.

Reply:
What's a reasonable amount?